cmd/rt-conf/main.go

```diff
@@ -13,17 +14,24 @@ import (
 )
 
 func main() {
-	configPath := flag.String("file",
+	if err := run(os.Args); err != nil {
+		log.Fatal(err)
```

Comment:
Since returned errors are not capitalized:
```suggestion
		log.Fatal("Error: ", err)
```

Reply:
Done! Thank you!